integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeWebApi.kt

```diff
@@ -15,15 +15,15 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package org.dash.wallet.integrations.crowdnode.ui
+package org.dash.wallet.integrations.crowdnode.api
 
-import androidx.fragment.app.Fragment
-import dagger.hilt.android.AndroidEntryPoint
-import org.dash.wallet.common.ui.viewBinding
-import org.dash.wallet.integrations.crowdnode.R
-import org.dash.wallet.integrations.crowdnode.databinding.FragmentPortalBinding
+import org.dash.wallet.integrations.crowdnode.model.CrowdNodeResponse
+import retrofit2.Response
+import retrofit2.http.*
 
-@AndroidEntryPoint
-class PortalFragment : Fragment(R.layout.fragment_portal) {
-    private val binding by viewBinding(FragmentPortalBinding::bind)
-}
\ No newline at end of file
+interface CrowdNodeWebApi {
+    @GET("odata/apifundings/GetFunds(address='{address}')")
```

Comment:
For now, we'll be using this endpoint, but perhaps in the future, there'll be something more suitable for the balance specifically.